pandas/core/resample.py

```diff
@@ -624,18 +624,134 @@ def backfill(self, limit=None):
 
     def fillna(self, method, limit=None):
         """
-        Fill missing values
+        Fill the new missing values in the resampled data using different
+        methods.
+
+        In statistics, imputation is the process of replacing missing data with
+        substituted values [1]_. When resampling data, missing values may
+        appear (e.g., when the resampling frequency is higher than the original
+        frequency).
+
+        The backward fill ('bfill') will replace NaN values that appeared in
+        the resampled data with the next value in the original sequence. The
+        forward fill ('ffill'), on the other hand, will replace NaN values
+        that appeared in the resampled data with the previous value in the
+        original sequence. Missing values that existed in the orginal data will
+        not be modified.
 
         Parameters
         ----------
         method : str, method of resampling ('ffill', 'bfill')
+            Method to use for filling holes in resampled data
+                * ffill: use previous valid observation to fill gap (forward
+                  fill).
+                * bfill: use next valid observation to fill gap (backward
+                  fill).
         limit : integer, optional
-            limit of how many values to fill
+            Limit of how many values to fill.
+
+        Returns
+        -------
+        Series, DataFrame
+            An upsampled Series or DataFrame with backward or forwards filled
+            NaN values.
+
+        Examples
+        --------
+
+        Resampling a Series:
+
+        >>> s = pd.Series([1, 2, 3],
+        ...               index=pd.date_range('20180101', periods=3, freq='h'))
+        >>> s
+        2018-01-01 00:00:00    1
+        2018-01-01 01:00:00    2
+        2018-01-01 02:00:00    3
+        Freq: H, dtype: int64
+
+        >>> s.resample('30min').fillna("bfill")
```

Comment:
I would maybe first show what it does without filling (which is `s.resample().asfreq()`), of course this is another method, but it will then be easier to see which values have actually been filled by `fillna()`